### ISSUE TITLE
v4.18.1 breaks API

### ISSUE THREAD

**Issue by ldng** [python-jsonschema/jsonschema#1123]
jsonschema.exceptions._RefResolutionError: 'bytes' object has no attribute 'timeout'

**Comment by Julian** [comment]
There's no information in your issue. Please follow up with a full description of your issue, including code to reproduce.

**Comment by seuf** [comment]
+1 v4.18.1 is breaking everything. I had to fix version to 4.18.0 in my requirements.
here is stack trace :
```
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/validators.py", line 1295, in validate
  error = exceptions.best_match(validator.iter_errors(instance))
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/exceptions.py", line 441, in best_match
  best = next(errors, None)
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/validators.py", line 359, in iter_errors
  for error in errors:
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/_validators.py", line 284, in ref
  yield from validator._validate_reference(ref=ref, instance=instance)
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/validators.py", line 452, in _validate_reference
  scope, resolved = resolve(ref)
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/validators.py", line 1076, in resolve
  return url, self._remote_cache(url)
File "/tmp/venv03txyqfk/lib/python3.8/site-packages/jsonschema/validators.py", line 1092, in resolve_from_url
  raise exceptions._RefResolutionError(exc)
onschema.exceptions._RefResolutionError: 'bytes' object has no attribute 'timeout'
```
